### DISCUSSION TITLE
Extension causes software crashes

### DISCUSSION THREAD

**Discussion by Johneltonsbrain** [protz/thunderbird-conversations#1629] [General]
When I click on the notification overlay that I get with a new Email, the whole software crashes since I have this extension installed.

**Comment by Standard8** [comment]
I would expect that this is generally an issue in Thunderbird itself, however it could be that Conversations is triggering something.

Please can you go to Help -> Troubleshooting Information, and look for the "Crash Reports for the Last 3 Days" section that is near the top.

Please copy some ids/links from there so we can have a look. There's an "All Crash Reports" link for if they were more than three days ago.

**Comment by Johneltonsbrain** [comment]
It looks like there wasnt a crash report created since Thunderbird just froze and not crash. Damnit.

https://imgur.com/a/XEAAOAd

I love this extension so much, I would hate having to disable it.
